docs/documentation/server_admin/topics/authentication/conditions.adoc

```diff
@@ -54,6 +54,15 @@ The sub-flow name to check if it was executed or not executed. Required.
 Check result:::
 When the condition evaluates to true. If `executed` returns true when the configured sub-flow was executed with output success, false otherwise. If `not-executed` returns false when the sub-flow was executed with output success, true otherwise (the negation of the previous option).
 
+`Condition - client scope`::
+The condition to evaluate if a configured client scope is present as a client scope of the client requesting authentication. These configuration fields exist:
+
+Client scope name:::
+The name of the client scope, which should be present as a client scope of the client, which is requesting authentication. If requested client scope is default client scope of the client requesting login, the condition will be evaluated to true. If requested client scope is optional client scope of the client requesting login, condition will be evaluated to true if client scope is sent by the client in the login request (EG. by the `scope` parameter in case of OIDC/OAuth2 client login). Required.
```

Comment:
```suggestion
The name of the client scope, which should be present as a client scope of the client, which is requesting authentication. If requested client scope is default client scope of the client requesting login, the condition will be evaluated to true. If requested client scope is optional client scope of the client requesting login, condition will be evaluated to true if client scope is sent by the client in the login request (for example, by the `scope` parameter in case of OIDC/OAuth2 client login). Required.
```